docs/illustration-guidelines.md

```diff
@@ -0,0 +1,52 @@
+# Primer Illustration Style Guide
+
+Guidelines for creating Illustrations for Primer
+
+# Symbols
+
+Primer illustrations are made up of a collection of various symbols which will be referred to as illustration components in order to maintain consistency between documentation and Figma. These components are meant to come together and represent the connection between Design + Code within the system. 
+
+The symbols used in Primer illustrations are:
+
+- Paintbrush
+- Squiggles
+- Dots
+- Geometric Shapes
+- Code Lines
+
+## Paintbrush
+
+The paintbrush represents the design/artistic side of Primer. It should only appear once per illustration.
+
+## Squiggles
+
+Squiggles come in two styles: 2px stroke/`blue-400` and 1px stroke/`blue-200`. They vary in size, but are typically 40x8 or 20x4 pixels. Squiggles may be oriented horizontally, vertically, or at a 45° angle.
+
+## Dots
+
+Dot patterns can be used to add some contrast and texture beneath elements. Currently we are only using the uniform dot grid as a drop shadow in illustrations. On dark backgrounds use `blue-700` and on light backgrounds `blue-200`.
+
+## Geometric Shapes
+
+Primer illustrations use 4 types of geometric shapes:
+
+- Circle
+- Square<sup><a href="#footnote-1">1</a></sup>
+- Triangle
+- Hexagon
+
+<sup><a name="footnote-1">1</a></sup> The square shape can be rotated 45° to register as a diamond as well.
+
+Shapes should only have a stroke and no fill. The stroke styles are 2px stroke/`blue-400` and 1px stroke/`blue-200`. When placing shapes next to one another the styling should be different.
+
+## Code Lines
+
+Code lines are represented by a row of rectangular blocks.
+
+**Dimensions**
+
+- Block width should be at a base-8 scale.  Block height should be limited to 16, 24, or 32 pixels.
```

Comment:
@shawnbot I wanted to be consistent with what we have on the [Primer style guide](https://styleguide.github.com/primer/support/spacing/#spacing-scale) which reads:

> The spacing scale is a base-8 scale. 

But since this is more about dimension than spacing I'm open to the suggestion. @broccolini what do you think?